api/tests/core/offerers/test_api.py

```diff
@@ -1058,6 +1058,45 @@ def test_create_offerer_on_known_offerer_by_user_rejected(self):
         assert actions_list[0].user == user
         assert actions_list[0].offerer == created_offerer
 
+    def test_create_offerer_on_known_offerer_twice(self):
+        # Given
+        user = users_factories.UserFactory()
+        offerer_informations = offerers_serialize.CreateOffererQueryModel(
+            name="Test Offerer", siren="418166096", address="123 rue de Paris", postalCode="93100", city="Montreuil"
+        )
+        offerer = offerers_factories.OffererFactory(siren=offerer_informations.siren)
+        offerers_factories.RejectedUserOffererFactory(user=user, offerer=offerer)
+
+        # When
+        updated_user_offerer = offerers_api.create_offerer(user, offerer_informations)
+
+        # Then
+        assert offerer.validationStatus == ValidationStatus.VALIDATED
+        assert updated_user_offerer.validationStatus == ValidationStatus.NEW
+        assert updated_user_offerer.dateCreated is not None
+
+        # When
+        updated_user_offerer = offerers_api.create_offerer(user, offerer_informations)
+
+        # Then
+        assert offerer.validationStatus == ValidationStatus.VALIDATED
+        assert updated_user_offerer.validationStatus == ValidationStatus.NEW
+        assert updated_user_offerer.dateCreated is not None
```

Comment:
Suggestion de simplification : 
```suggestion
        for _ in range(2):
            # When
            updated_user_offerer = offerers_api.create_offerer(user, offerer_informations)

            # Then
            assert offerer.validationStatus == ValidationStatus.VALIDATED
            assert updated_user_offerer.validationStatus == ValidationStatus.NEW
            assert updated_user_offerer.dateCreated is not None
```